docs/basics/101-135-help.rst

```diff
@@ -0,0 +1,145 @@
+.. _help:
+
+How to get help
+---------------
+
+After all of the DataLad-101 lectures and tutorials so far,
+you really begin to appreciate the pre-crafted examples the
+lecturer provided the students with.
+
+"Nothing really goes wrong, and if so, it's intended", you
+acknowledge. "But how does this prepare me for life after
+the course? I've seen a lot of different errors and know many
+caveats and principles already, but I certainly will mess
+something up at one point, and how will I get help then?"
+
+Fortunately, the syllabus has a whole section on that, and on
+one lazy, warm summer-afternoon you flip through it.
+
+You realize that you already know the most important things:
+The number one advice on how to get help is
+`"Read the error message." <http://poster.keepcalmandposters.com/default/5986752_keep_calm_and_read_the_error_message.png>`_.
+The second advice it
+`"I'm not kidding: Read the error message <https://images.app.goo.gl/GWQ82AAJnx1dWtWx6>`_.
+The third advice, finally, says
+`"Honestly, read the f***ing error message <https://images.app.goo.gl/ddxg4aowbji6XTrw7>`_.
+
+Help yourself
+^^^^^^^^^^^^^
+
+If you run into a DataLad problem and you have followed the three
+steps above, but the error message
+`doesn't give you a clue on how to proceed <https://imgs.xkcd.com/comics/code_quality_3.png>`_,
+the first you should do is
+
+#. find out which *version* of DataLad you use
+
+#. read the *help page* of the command that failed
+
+The first step is important in order to find out whether a
+command failed due to using a wrong DataLad version. In order
+to use this book and follow along, your DataLad version
+should be ``datalad-0.12`` or higher, for example.
+
+To find out which version you are using, run
+
+.. runrecord:: _examples/DL-101-135-101
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ datalad --version
+
+.. index:: ! datalad command; wtf
+
+If you want a comprehensive overview of your full setup,
+:command:`datalad wtf` is the command to turn to
+(:manpage:`datalad-wtf` manual). Running this command will
```

Comment:
Offer translation help. It could mean:

- Why the Face?
- Wow, That's Fantastic
- What's This For
- What To Fix
- Waste Treatment Facility	
- What the FAQ?
- ...

Reply:
To be honest, I don't know what ``wtf`` stands for. In my head, I happily say "datalad what-the-fuck". What does it *actually* stand for?

Reply:
You hit the nail on the head ;-)

Reply:
Haha ;-) I like how versatile this abbreviation is.